pkg/standalone/standalone.go

```diff
@@ -197,7 +206,7 @@ func getDownloadDest(installLocation string) (string, error) {
 
 // installLocation is not used, but it is present because it's required to fit the initSteps func above.
 // If the number of args increases more, we may consider passing in a struct instead of individual args.
-func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string) {
+func runRedis(wg *sync.WaitGroup, errorChan chan<- error, dir, version string, dockerNetwork string, installLocation string, _ string) {
```

Comment:
Do we need to run redis container when redisHost is given - we won't be using this container anyways right?

Reply:
👍

Reply:
You mean if the redis-host != localhost? But what if the user specifies it as 127.0.0.1 or specifies the IP of the machine itself? redis-host != localhost might not be good enough to decide that

Reply:
As discussed offline, if the user specifies a redis-host, the user will need to make sure it is running (even if the user has specified the same machine)